pom.xml

```diff
@@ -33,6 +33,11 @@
       <artifactId>appengine-api-1.0-sdk</artifactId>
       <version>1.9.59</version>
     </dependency>
+    <dependency>
```

Comment:
Did you mean to add this in this PR?

Reply:
I didn't! Must have happened when I pulled from master. Will delete the extra line now.

Reply:
Sorry -- I misunderstood the question. pom.xml was untracked in my previous PR so I had to add it here. Hope that's okay.